src/test/java/gov/usgs/earthquake/aws/JsonNotificationTest.java

```diff
@@ -0,0 +1,43 @@
+package gov.usgs.earthquake.aws;
+
+import java.net.URL;
+
+import javax.json.Json;
+
+import org.junit.Test;
+
+import gov.usgs.earthquake.product.Product;
+import gov.usgs.earthquake.product.io.JsonProduct;
+import gov.usgs.util.StreamUtils;
+import gov.usgs.util.protocolhandlers.data.Handler;
+
+public class JsonNotificationTest {
+  static {
+    Handler.register();
+  }
+
+  public static final String JSON_NOTIFICATION_URL = "data:;base64,"
```

Comment:
This needs to be purged from the commit history. It contains the S3 hostname for a private development bucket.

Reply:
purge complete